command/agent/secure_variable_endpoint.go

```diff
@@ -0,0 +1,109 @@
+package agent
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+func (s *HTTPServer) SecureVariablesListRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	if req.Method != "GET" {
+		return nil, CodedError(405, ErrInvalidMethod)
+	}
+
+	args := structs.SecureVariablesListRequest{}
+	if s.parse(resp, req, &args.Region, &args.QueryOptions) {
+		return nil, nil
+	}
+
+	var out structs.SecureVariablesListResponse
+	if err := s.agent.RPC("SecureVariables.List", &args, &out); err != nil {
+		return nil, err
+	}
+
+	setMeta(resp, &out.QueryMeta)
+
+	if out.Data == nil {
+		out.Data = make([]*structs.SecureVariableStub, 0)
+	}
+	return out.Data, nil
+}
+
+func (s *HTTPServer) SecureVariableSpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	path := strings.TrimPrefix(req.URL.Path, "/v1/var/")
+	if len(path) == 0 {
+		return nil, CodedError(400, "Missing secure variable path")
+	}
+	switch req.Method {
+	case "GET":
```

Comment:
There are consts for these like `http.MethodGet` in `net/http`: https://pkg.go.dev/net/http#pkg-constants but we're admittedly not very good about remembering to use them 😀